### DISCUSSION TITLE
Direct link rendered image - use of external_image_storage

### DISCUSSION THREAD

**Discussion by lgogolin** [grafana/grafana#60386] [General]
It would be really nice and beneficial if `Direct link rendered image` could re-use `external_image_storage` configuration.

Flow that I had in mind for this would be:
- Click on `Direct link rendered image`
- That push rendered image to `external_image_storage`
- In return we are getting img url from external storage

Currently `Direct link rendered image` is basically useless:
- It's impossible to share image of an panel in communicators like: Slack 
- To allow accessing `Direct link rendered image` grafana have to be publicly available and whats even worse: `auth.anonymous` would have to be enable - which is unacceptable in production enviroments

**Comment by mellieA** [comment]
This discussion was converted into issue https://github.com/grafana/grafana/issues/75324
